src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUES_OF_SIDE = 20;
+    private static final int MAX_RADIUS = 15;
+    private static final int UNIQUE_RADIUS = 10;
+    private static final int VALUES_OF_FIGURES = 5;
+
+    private ColorSupplier color = new ColorSupplier();
+    private Random random = new Random();
+    private Figure circleDefault = new Circle(UNIQUE_RADIUS, Color.WHITE.name());
```

Comment:
It’s better not to use it like that, but simply return a new figure with default parameters